Lusas_oM/Settings/LusasSettings.cs

```diff
@@ -21,14 +21,19 @@
  */
 
 using BH.oM.Base;
+using System.ComponentModel;
+using BH.oM.Adapter;
 
 namespace BH.oM.Adapters.Lusas
 {
-    public class LusasConfig : BHoMObject
+    public class LusasSettings : AdapterSettings
     {
         /***************************************************/
         /****            Public Properties              ****/
         /***************************************************/
+ 
+        [Description("Sets the merging tolerance in Lusas and the distance comparers used by the Lusas_Adapter.")]
```

Comment:
This description suggests you can alter the comparer used in the Lusas_Adapter - the code is found here:
https://github.com/BHoM/Lusas_Toolkit/tree/develop/Lusas_Engine/Objects/Equality_Comparer

I would either update the description or implement the changes to the comparer.

Reply:
Thank you. Changed only in the description for now.